esphome/components/adc/adc_sensor.h

```diff
@@ -31,6 +31,7 @@ class ADCSensor : public sensor::Sensor, public PollingComponent, public voltage
   /// `HARDWARE_LATE` setup priority.
   float get_setup_priority() const override;
   void set_pin(InternalGPIOPin *pin) { this->pin_ = pin; }
+  void set_raw(bool output_raw) { output_raw_ = output_raw; }
```

Comment:
Usually the setter is named after the variable:
```suggestion
  void set_output_raw(bool output_raw) { output_raw_ = output_raw; }
```